ludwig/schema/features/preprocessing/category.py

```diff
@@ -113,3 +114,34 @@ def __post_init__(self):
     )
 
     vocab: List[str] = schema_utils.List(default=None)
+
+
+@DeveloperAPI
+@register_preprocessor("category_llm")
+@ludwig_dataclass
+class LLMCategoryOutputPreprocessingConfig(CategoryOutputPreprocessingConfig):
+    def __post_init__(self):
+        if self.labels is None:
+            raise ConfigValidationError("`labels` must be specified for `category_llm` output feature.")
+        if self.fallback_label is None:
+            raise ConfigValidationError("`fallback_label` must be specified for `category_llm` output feature.")
+
+    labels: List[str] = schema_utils.List(default=None)
+
+    fallback_label: str = schema_utils.String(
+        default="",
+        allow_none=False,
+        description="The label to use when the model doesn't match any of the labels in the `labels` list.",
+    )
+
+    prompt_template: str = schema_utils.String(
+        default="",
+        allow_none=False,
+        description="The template to use for the prompt. The labels will be inserted into the template.",
+    )
+
+    is_llm_task: bool = schema_utils.Boolean(
+        default=True,
+        description="Whether the model is a language modeling task or not.",
```

Comment:
semantics here are a bit meh. a model is not a task. what is this trying to capture?

Reply:
Got rid of it, don't need it anymore